Utilities/utilities.css

```diff
@@ -0,0 +1,76 @@
+/* reset css */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+a {
```

Comment:
In place of using an element selector, try using a class as it will affect all the anchor tags.